src/Mayaqua/Network.h

```diff
@@ -147,6 +147,9 @@ struct SSL_ACCEPT_SETTINGS
 	bool Tls_Disable1_0;
 	bool Tls_Disable1_1;
 	bool Tls_Disable1_2;
+	bool Tls_Disable1_3;
+	bool Override_Security_Level;
+	UINT32 Override_Security_Level_Value;

```

Comment:
`UINT32` is not defined anymore since #1195, it should be `UINT` instead.

Sorry about that, this PR should've been merged earlier.